docs/edd-request-form.html

```diff
@@ -141,23 +141,27 @@
         </div>
         <div class="nypl-row">
         <div class="nypl-column-one-quarter">
-          <div class="nypl-text-field" id="article-title">
-            <label for="required-field" id="article-title-label">Starting Page:
+          <div class="nypl-text-field" id="article-start-page">
+            <label for="required-start-page-field" id="article-start-page-label">Starting Page:
               <span class="nypl-required-field">Required</span>
             </label>
-            <input id="required-field" type="text" required="true" aria-labelledby="article-title-label article-title-status" aria-required="true" placeholder="" />
-            <!-- <span class="nypl-field-status" id="article-title-status" aria-live="assertive" aria-atomic="true">Please indicate the title of the chapter or article </span> -->
+
+            <input id="required-start-page-field" type="text" required="true" aria-labelledby="article-start-page-label article-start-page-status" aria-required="true" placeholder="" />
+
+            <span class="nypl-field-status" id="article-start-page-status" aria-live="assertive" aria-atomic="true">Values must be numeric only (No special characters allowed)</span>
           </div>
-          <div class="nypl-text-field" id="article-title">
-            <label for="required-field" id="article-title-label">Ending Page:
+          <div class="nypl-text-field" id="article-end-page">
+            <label for="required-end-page-field" id="article-end-page-label">Ending Page:
               <span class="nypl-required-field">Required</span>
             </label>
-            <input id="required-field" type="text" required="true" aria-labelledby="article-title-label article-title-status" aria-required="true" placeholder="" />
-            <span class="nypl-field-status" id="article-title-status" aria-live="assertive" aria-atomic="true">Values must be numeric only (No special characters allowed)</span>
```

Comment:
So Denise mentioned  that Princeton's platform doesn't need even a chapter title, while it does need in the API, they just put "1" when the patron didn't indicate anything.

We might be able to do it too, but I don't know if it will cause any issues in the API or it is really what the patron wants.